pkg/build/builder/common_test.go

```diff
@@ -32,15 +33,12 @@ func TestBuildInfo(t *testing.T) {
 						},
 					},
 				},
-				Revision: &api.SourceRevision{
-					Git: &api.GitSourceRevision{
-						Commit: "1575a90c569a7cc0eea84fbd3304d9df37c9f5ee",
-					},
-				},
 			},
 		},
 	}
-	got := buildInfo(b)
+	sourceInfo := &git.SourceInfo{}
+	sourceInfo.CommitID = "1575a90c569a7cc0eea84fbd3304d9df37c9f5ee"
+	got := buildInfo(b, sourceInfo)
```

Comment:
Should you not test both paths? (one with the revision in the build and the other one with sourceInfo?)